### ISSUE TITLE
Cant extract Mario Kart 8 Update 2.0.0 nsp

### ISSUE THREAD

**Issue by CodeF53** [jakcron/nstool#80]
I am on windows using NSTool v1.6.3,

This is the command I am using:
```
nstool.exe -k prod.keys --fstree "MK8 UPDATE 2.0.0.nsp"
```
This is thew error I get
```
[nn::hac::PartitionFsMetaGenerator ERROR] sPfsFile entry had unexpected data offset.
```

**Comment by jakcron** [comment]
That error occurs when the NSP underlaying PFS0 structure wasn't correct.

What tool was that NSP generated with?

**Comment by jakcron** [comment]
This should be fixed in v1.6.4.

Please download a copy from the release page.

https://github.com/jakcron/nstool/releases/tag/v1.6.4

**Comment by CodeF53** [comment]
Thanks!
